docs/commands/teardown-registry.md

```diff
@@ -0,0 +1,9 @@
+Tear Down Private Registry
+===============================================================================
+
```

Comment:
Can we add some more words to this file. What does it do? Why would you use it? Any impacts of doing this? Will running this cause my MAS system to stop working? Do I have to mirror again?